src/test/resources/features/IFramePrectice.feature

```diff
@@ -12,8 +12,6 @@ Feature: iFrame handling
     When user enters the value in input iframe
     Then user should be seeing entered value
 
-  @iframe_handling2
-  Scenario: iFrame handling
-    Given user launches the browser with new url
-    When user enters the value in input iframe
-    Then user should be seeing entered value
+    @feature_handling
```

Comment:
Hi @ritusolanki8  ,

Can we know why are you removed the test case

Reply:
Changes have been addressed, please review..